.devcontainer/devcontainer.json

```diff
@@ -26,8 +26,8 @@
 
   "extensions": ["dbaeumer.vscode-eslint", "esbenp.prettier-vscode", "Orta.vscode-jest"],
 
-  // Run `yarn install` the moment after the container has finished starting, but before the extension starts loading.
-  "onCreateCommand": "yarn install",
+  // Run `pnpm install` the moment after the container has finished starting, but before the extension starts loading.
+  "onCreateCommand": "npm install -g pnpm && pnpm install",
```

Comment:
- I really want to install pnpm in `.devcontainer/Dockerfile`.
- However, it doesn't work because it is installed in a location where the PATH is not set.
- So I use npm to install pnpm at the time of `onCreateCommand`.